modules/mass.js

```diff
@@ -78,7 +78,7 @@ export function updateRenderers() {
 /**
  * Before hook for 'addBidResponse'.
  */
-export function addBidResponseHook(next, adUnitCode, bid) {
+export function addBidResponseHook(next, adUnitCode, bid, {index = auctionManager.index} = {}) {
```

Comment:
@snapwich It seems like this new `index` param doesn't exist on the original hooked function's call signature (see in the auction.js file).  

By adding this new variable here, would that cause any abnormalities with the underlying hook workflow?  I'm guessing from how this is used here (ie that it's totally local within this function), it may be fine - but wanted to double-check.